nemo/collections/asr/data/audio_to_text_lhotse_prompted.py

```diff
@@ -102,126 +105,5 @@ def _collate_tokens(self, tokens: list[list[int]]) -> tuple[torch.Tensor, torch.
         return tokens, token_lens
 
 
-# Mapping from a string name to a known prompt formatter function.
```

Comment:
Necessary refactor: split into two files and moved to `common/prompts/{fn,canary}.py`. Having it there allows us to measure the actual output sequence length using the complete tokenized + prompt formatted transcript before we put the training examples into buckets.